### PR TITLE
Features layout

### PR DESCRIPTION
**Pull request by andrestobelem**
# Summary
* I added a layout for the features. 
* I couldn't make a border top with an image.
* I used flexbox to wrap elements when resizing the browser

# Trello Card

https://trello.com/c/WhjAGV05

### CODE REVIEW

File: app/assets/stylesheets/commons.scss [hunk 1]
```diff
@@ -6,9 +6,45 @@
...
+  height: 12px;
+}
+
+.features {
+  display: flex;
+  background-color: $gray-bg;
+  min-height: 288px;
+  align-items: center;
+  justify-content: space-around;
+  flex-flow: row wrap;
```
Comment (SKOLZ) [inline app/assets/stylesheets/commons.scss]: row is the default flex-direction so if you want to wrap it's content you should use ```flex-wrap: wrap``` , flex flow is used when you want to wrap and change the direction to column for example.

File: app/assets/stylesheets/commons.scss [hunk 2]
```diff
@@ -6,9 +6,45 @@
...
+}
+
+.feature {
+  width: 200px;
+  font: Myriad Pro;
```
Comment (SKOLZ) [inline app/assets/stylesheets/commons.scss]: define the font in a less specific way, like this for example:
```scss
* {
  font-family: 'Myriad Pro', Tahoma;
}
```
you should also use font-family for this and you should use a fallback font and wrap the custom one between quotes like the previous example.

File: app/views/landing/index.html.slim [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+    = image_tag 'separator_bar.png', class: 'landing-image hidden-xs hidden-sm'
+    = image_tag 'tcg_logo.png', class: 'landing-image', alt: 'The cheese guardian'
+
+  .horizontal-saw
+
+  .features
+
+    .feature
+      = image_tag 'ic_list.png', class: 'feature-image'
+      p.feature-title
```
Comment (SKOLZ) [inline app/views/landing/index.html.slim]: the idea of html tags is to have some default style if the css does not load, so the feature title should use a headline tag for example an h1 in this case as it's a title.

File: app/views/landing/index.html.slim [hunk 4]
```diff
@@ -1,4 +1,33 @@
...
-  = image_tag 'separator_bar.png', class: 'landing-image hidden-xs hidden-sm'
-  = image_tag 'tcg_logo.png', class: 'landing-image', alt: 'The cheese guardian'
+.container
+
+  .main.full-height
+    = image_tag 'phone.png', class: 'landing-image hidden-xs hidden-sm'
+    = image_tag 'separator_bar.png', class: 'landing-image hidden-xs hidden-sm'
+    = image_tag 'tcg_logo.png', class: 'landing-image', alt: 'The cheese guardian'
+
+  .horizontal-saw
```
Comment (SKOLZ) [inline app/views/landing/index.html.slim]: you could put the background of this div as the background of the ```.main``` div and keep a simple structure. Although that would be a better approach you would be adding style related to the ```.features``` div to a div which has nothing to do with it so i would suggest you add it as a ```::before``` element in the ```.features``` css, if you need i can help you with this

Comment (andrestobelem) [inline app/views/landing/index.html.slim]: Yes, I will ask for help here.

**Pull request merged**